drivers/clock_control/stm32_ll_clock.c

```diff
@@ -223,7 +236,9 @@ static int stm32_clock_control_init(struct device *dev)
 
 	/* Disable other clocks */
 	LL_RCC_HSE_Disable();
+#ifndef CONFIG_SOC_SERIES_STM32F0X
```

Comment:
Remove the #ifdef here, as you defined LL_RCC_MSI_Disable in stm32_f0_ll_clock.c file
It helps reducing the use of ifdef in this generic file (when doable)